mllib/src/test/scala/org/apache/spark/ml/regression/RandomForestRegressorSuite.scala

```diff
@@ -28,6 +28,7 @@ import org.apache.spark.mllib.tree.configuration.{Algo => OldAlgo}
 import org.apache.spark.rdd.RDD
 import org.apache.spark.sql.{DataFrame, Row}
 
+
```

Comment:
Not a big deal but if you're revising this, you could revert whitespace changes like this

Reply:
good catch!  done!  sorry about that